scripts/basic-workflow-tests.sh

```diff
@@ -508,21 +512,20 @@ wait_for_server() {
 usage() {
     echo "Usage: $0 [-m] [-h]"
     echo "  -e  Run model evaluation"
-    echo "  -T  Use the 'full' training library rather than legacy training"
```

Comment:
@cdoern you will need to update the Training Library CI as well - this is a breaking change: https://github.com/instructlab/training/blob/main/.github/workflows/e2e-nvidia-t4-x1.yml#L137

Reply:
cc @RobotSail @Maxusmusti @JamesKunstle FYI

Reply:
will do

Reply:
please link here once you have, thanks!

Reply:
https://github.com/instructlab/training/pull/222